08-rails/installing-foundation.md

```diff
@@ -0,0 +1,120 @@
+#  Installing Foundation By Default
+
+## Learning Goals
+- Practice modifying our `.rails-template.rb` file to add an additional gem and content.
+- Learn how to install and use the Foundation Framework via the gem.
+
+## Installing the Foundation Gem
+
+To install Foundation in your Rails project via the Gem first add the following to your Gemfile.
+
+```ruby
+gem 'foundation-rails'
+```
+
+And run `bundle install` from the terminal in your rails project.  
+
+```bash
+$  bundle install
```

Comment:
I think by this point we probably don't need a separate code snippet to explain how to run `bundle install`.